system/jlib/jexcept.cpp

```diff
@@ -1117,6 +1117,10 @@ NO_SANITIZE("alignment") void excsighandler(int signum, siginfo_t *info, void *e
     if (nested++)
         return;
 
+    //If the program is terminating then do not try and trace
+    if (!queryLogMsgManager())
+        return;
+
```

Comment:
Could only perhaps check if stderr is valid and switch logging to that but I don't think its worth it.

Reply:
I think stderr is likely to be invalid by this point as well.  I was hitting a crash in ~std::locale